easybuild/tools/module_generator.py

```diff
@@ -210,7 +210,7 @@ def get_description(self, conflict=True):
         """
         Generate a description.
         """
-        description = "%s - Homepage: %s" % (self.app.cfg['description'], self.app.cfg['homepage'])
+        description = "Description: %s - Homepage: %s" % (self.app.cfg['description'], self.app.cfg['homepage'])
 
```

Comment:
Why not introduce the 'Description: ' prefix only for the `whatis` text (i.e., 3 lines below)?  For the `module help` output it should be pretty clear that this is a description...